src/app/core/models/github-user.model.ts

```diff
@@ -11,3 +16,55 @@ export interface GithubUser {
   updated_at: string;
   url: string;
 }
+
+/**
+ * Represents a GitHub user in WATcher
+ */
+export class GithubUser implements Group {
+  static NO_ASSIGNEE: GithubUser = new GithubUser({
+    avatar_url: '',
+    created_at: '',
+    html_url: '',
+    login: 'Unassigned',
+    name: '',
+    node_id: '',
+    two_factor_authentication: false,
+    site_admin: false,
+    type: '',
+    updated_at: '',
+    url: ''
+  } as RawGithubUser);
```

Comment:
I don't think this type casting is necessary? As long as the object has all the keys, there is no need to do type casting.